src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/App.tsx

```diff
@@ -21,6 +21,8 @@ export const App = () => {
   const dispatch = useDispatch();
   const hasErrorSelector = makeGetHasErrorsSelector();
   const hasApiErrors: boolean = useSelector(hasErrorSelector);
+  // @ts-ignore since appliacationSetting is not available in rootstate
```

Comment:
I'm not sure this comment is needed anymore after merge from master - can you merge from master, remove this comment, and see if it still complains?  There are also a couple conflicts that needs to be handled. Let me know if you get stuck on this.

If we still need this comment, you also need to add a new comment above it to ignore the ts-ignore comment, like this: 

```typescript
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore since appliacationSetting is not available in rootstate
const appOidcProvider = useSelector((state) => state.applicationSettings?.applicationSettings?.appOidcProvider);
```

We could add the missing typings manually, but I'm planning a PR to get automatic typings for store state anyway, so IMO its fine to ignore this for now.